R/utils.R

```diff
@@ -1138,11 +1150,12 @@ markdown_to_latex <- function(text, md_engine) {
 
 # Transform Markdown text to ooxml
 markdown_to_xml <- function(text) {
+
   res <- vapply(
-    as.character(text),
+    gsub("<br>","\n\n",as.character(text)),
     FUN.VALUE = character(1L),
     USE.NAMES = FALSE,
-    FUN = commonmark::markdown_xml
+    FUN = \(x, ...) commonmark::markdown_xml(linebreak_br(x), ...)
```

Comment:
```suggestion
    FUN = function(x, ...) commonmark::markdown_xml(linebreak_br(x), ...)
```

not sure how compatible we are with R < 4.1, but let's not use this syntax just now.

Reply:
makes sense!